0002-metric-measure.md

```diff
@@ -0,0 +1,41 @@
+# Replace Raw statistics with Measure-type Metric
+
+Define a new Metric type named "Measure" to cover existing "Raw" statistics uses.
+
+## Motivation
+
+The primary motivation is that Raw statistics should support the optimization and usability improvements associated with pre-defined label values (0001-metric-pre-defined-labels).  By elevating non-Cumulative, non-Gauge statistics to the same conceptual level as Metrics in the API, we effectively make the type of a metric independent from whether it supports pre-defined labels.
+
+This also makes it possible to eliminate the low-level `stats.Record` interface from the API specification entirely (0003-eliminate-stats-record).
+
+## Explanation
+
+This proposal suggests we think about which aggregations apply to a metric independently from its type.  A MeasureMetric could be used to aggregate a Histogram, or a Summary, or _both_ of these aggregations simultaneously.  This proposal makes metric type independent of aggregation type, whereas there is a precedent for combining these types into one.
```

Comment:
I think I confused you a bit because we did not define the Histograms/Summaries see https://github.com/open-telemetry/opentelemetry-specification/issues/146.

The idea is the following, we offer support for users to record metrics where the developer that instruments the code can choose between exporting a pre-defined aggregation/label metrics (`type A`) or a "raw statistics" (`type B`) (no aggregation, labels pre-defined). The way how this decision is made tries to follow this rules:
1. If the metric is pre-aggregated (e.g. CPU usage is a counter in /proc) and no labels (here I mean that there are no labels used to break down this metric in /proc, but constant labels can be added to this metric if necessary. Then a `type A` metric should be used because we cannot apply any other aggregation (if you want to calculate rate which is possible for any counter (monotonic increasing metric) that can be done in all modern backends Prometheus, Stackdriver, etc.).
2. If the metric is recorded with every request (rpc latency, bytes received, bytes send) and the developer of the instrumentation is not the application owner (third-party library like OpenTelemetry itself) then `type B` should be used. This way the application owner can decide how to aggregate these metrics.
3. If the metric is a simple metric that does not make sense to aggregate in a different way (e.g. queue length - this is a gauge and probably does not require any labels to be used to breakdown) then a `type A` metric should be used.